README.md

```diff
@@ -1 +1,106 @@
-# c11-apollo-bandcamp-tracker
+# 🎸 Bandcamp Data Pipeline
```

Comment:
Maybe "Apollo Bandcamp Project" would be a better title since the project doesn't just include a pipeline